package.json

```diff
@@ -12,8 +12,8 @@
   "scripts": {
     "test": "mocha -r ts-node/register tests/**/*.spec.ts",
     "lint": "resin-lint --typescript lib examples tests",
-    "build": "tsc",
-    "install": "tsc"
+    "build": "rimraf build && tsc",
+    "install": "npm run build"
```

Comment:
If this runs `rimraf` on install, then `rimraf` can't be a devDependency ;)

Reply:
It is a devDependency already.

Reply:
But it shouldn't – the `install` step is run when installing it as a dependency, and in that case devDependencies are not installed

Reply:
Oh sorry, I've misread your comment, updated.